### PR TITLE
Bug 1859597 - updated Stub URI parser arguments to include windows_ubr and stub_build_id

### PR DESCRIPTION
**Pull request by erchen03**
for [Bug 1859597](https://bugzilla.mozilla.org/show_bug.cgi?id=1859597)
related pull request: https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/829

### CODE REVIEW

File: ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java [hunk 1]
```diff
@@ -288,7 +288,8 @@ private static class InvalidIntegerException extends InvalidUriException {
         putBoolPerCode(ImmutableMap.of("set_default", 2)), //
         putString("download_ip"), putString("attribution"),
         putIntegerAsString("profile_cleanup_prompt"), putBool("profile_cleanup_requested"),
-        putString("distribution_id"), putString("distribution_version"));
+        putString("distribution_id"), putString("distribution_version"),
+        putInteger("windows_ubr"), putString("stub_build_id"));
```
Comment (BenWu) [inline ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java:292]: I think this should fix the CI error
```suggestion
        putString("distribution_id"), putString("distribution_version"), putInteger("windows_ubr"), 
        putString("stub_build_id"));
```

**Review by BenWu — APPROVED** [review]
This looks fine to me and it should be safe to merge.  @akkomar do you want to double-check just in case?

File: ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java [hunk 2]
```diff
@@ -288,7 +288,8 @@ private static class InvalidIntegerException extends InvalidUriException {
         putBoolPerCode(ImmutableMap.of("set_default", 2)), //
         putString("download_ip"), putString("attribution"),
         putIntegerAsString("profile_cleanup_prompt"), putBool("profile_cleanup_requested"),
-        putString("distribution_id"), putString("distribution_version"));
+        putString("distribution_id"), putString("distribution_version"), putInteger("windows_ubr"),
+        putString("stub_build_id"));
```
Comment (akkomar) [inline ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java:292]:
```suggestion
        putString("distribution_id"), putString("distribution_version"), //
        putInteger("windows_ubr"), putString("stub_build_id"));
```

Comment (akkomar) [reply ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java:292]: I'd put them on a single line so it's easier to track history of this file.

**Review by akkomar — APPROVED** [review]

**Pull request merged**